### PR TITLE
Add Digital Ocean release instructions

### PR DESCRIPTION
**Pull request by hardillb**
Add details of how to publish the Digital Ocean Droplet release once it's been built as part of a release.

### CODE REVIEW

**Comment by hardillb** [comment]
Don't merge before https://github.com/flowforge/website/pull/747 I will rebase afterwards

File: src/handbook/development/releases/digital-ocean.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+navTitle: Release Process - Digital Ocean
+---
```
Comment (ZJvandeWeg) [inline src/handbook/development/releases/digital-ocean.md:3]:
```suggestion
---

```

File: src/handbook/development/releases/digital-ocean.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+# Digital Ocean Release Process
+
+When a release is tagged in the [Digital Ocean](https://github.com/flowforge/digital-ocean) 
+repository a GitHub Action will run and rebuild the image.
+
+Once complete the Release Manager should post to the #dev slack chanel to ask somebody with
+Digital Ocean access to do the following steps.
+
+ 1. Log into the FlowForge team on Digital Ocean
```
Comment (ZJvandeWeg) [inline src/handbook/development/releases/digital-ocean.md:12]: Are the credentials in 1Password?

**Comment by ZJvandeWeg** [comment]
@hardillb approval pending this PR not being a draft.

**Pull request merged**